config/locales/en/members.yml

```diff
@@ -337,7 +337,7 @@ en:
 
     - first_name: 'Noah'
       last_name: 'Alcus'
-      email: 'averyyip@berkeley.edu'
+      email: 'ntalcus@berkeley.edu'
```

Comment:
That said, it would be nice if I could add my website info as well B)
ntalcus.github.io

Reply:
i don't think the website links are being used lmao
but i included it anyway